Pinta.Core/Effects/ColorBgra.cs

```diff
@@ -207,6 +207,27 @@ public static ColorBgra FromUInt32 (uint bgra)
 		return color;
 	}
 
+	/// <summary>
+	/// Performs Alpha Blending between two colors. In practice, Alpha Blending two images produces the same result
+	/// as layering the top image on top of the bottom image.
+	/// </summary>
+	/// <param name="colorTop">The color layered on top. Color must be in premultiplied form. See <see cref="ToPremultipliedAlpha"/>.</param>
+	/// <param name="colorBottom">The color layered below. Color must be in premultiplied form. See  <see cref="ToPremultipliedAlpha"/>.</param>
+	/// <returns>Premultiplied form of the alpha-blend between colorTop and colorBottom.</returns>
+	public static ColorBgra AlphaBlend (ColorBgra colorTop, ColorBgra colorBottom)
```

Comment:
I think it should also be possible to do this blending using the `NormalBlendOp` class, so I'd suggest giving that a try!

Reply:
Done. Removed the alphablend method while i was at it